cypress-run.js

```diff
@@ -0,0 +1,7 @@
+const {execSync} = require('child_process');
+const uploadRecordings = process.env.npm_package_config_cypressUploadRecordings;
+let command = "npx cypress run";
+if(uploadRecordings) {
+    command += ` --record --key ${process.env.CYPRESS_RECORD_KEY}`
```

Comment:
According with the documentation, this is not required.
https://docs.cypress.io/guides/guides/command-line.html#cypress-run-record-key-lt-record-key-gt
We can check the existence of the env and throw an error

Reply:
Done!